src/code/z_parameter.c

```diff
@@ -2668,39 +2668,47 @@ void Interface_DrawItemButtons(GlobalContext* globalCtx) {
+                       R_ITEM_BTN_DD(HUD_ITEM_BTN_B) * 2, R_ITEM_BTN_DD(HUD_ITEM_BTN_B) * 2);
 
     // C-Left Button Color & Texture
     gDPPipeSync(OVERLAY_DISP++);
     gDPSetPrimColor(OVERLAY_DISP++, 0, 0, R_C_BTN_COLOR(0), R_C_BTN_COLOR(1), R_C_BTN_COLOR(2),
                     interfaceCtx->cLeftAlpha);
-    gSPTextureRectangle(OVERLAY_DISP++, R_ITEM_BTN_X(1) << 2, R_ITEM_BTN_Y(1) << 2,
-                        (R_ITEM_BTN_X(1) + R_ITEM_BTN_WIDTH(1)) << 2, (R_ITEM_BTN_Y(1) + R_ITEM_BTN_WIDTH(1)) << 2,
-                        G_TX_RENDERTILE, 0, 0, R_ITEM_BTN_DD(1) * 2, R_ITEM_BTN_DD(1) * 2);
+    gSPTextureRectangle(OVERLAY_DISP++, R_ITEM_BTN_X(HUD_ITEM_BTN_C_LEFT) << 2, R_ITEM_BTN_Y(HUD_ITEM_BTN_C_LEFT) << 2,
+                        (R_ITEM_BTN_X(HUD_ITEM_BTN_C_LEFT) + R_ITEM_BTN_WIDTH(HUD_ITEM_BTN_C_LEFT)) << 2,
+                        (R_ITEM_BTN_Y(HUD_ITEM_BTN_C_LEFT) + R_ITEM_BTN_WIDTH(HUD_ITEM_BTN_C_LEFT)) << 2,
+                        G_TX_RENDERTILE, 0, 0, R_ITEM_BTN_DD(HUD_ITEM_BTN_C_LEFT) * 2,
+                        R_ITEM_BTN_DD(HUD_ITEM_BTN_C_LEFT) * 2);
 
     // C-Down Button Color & Texture
     gDPSetPrimColor(OVERLAY_DISP++, 0, 0, R_C_BTN_COLOR(0), R_C_BTN_COLOR(1), R_C_BTN_COLOR(2),
                     interfaceCtx->cDownAlpha);
-    gSPTextureRectangle(OVERLAY_DISP++, R_ITEM_BTN_X(2) << 2, R_ITEM_BTN_Y(2) << 2,
-                        (R_ITEM_BTN_X(2) + R_ITEM_BTN_WIDTH(2)) << 2, (R_ITEM_BTN_Y(2) + R_ITEM_BTN_WIDTH(2)) << 2,
-                        G_TX_RENDERTILE, 0, 0, R_ITEM_BTN_DD(2) * 2, R_ITEM_BTN_DD(2) * 2);
+    gSPTextureRectangle(OVERLAY_DISP++, R_ITEM_BTN_X(HUD_ITEM_BTN_C_DOWN) << 2, R_ITEM_BTN_Y(HUD_ITEM_BTN_C_DOWN) << 2,
+                        (R_ITEM_BTN_X(HUD_ITEM_BTN_C_DOWN) + R_ITEM_BTN_WIDTH(HUD_ITEM_BTN_C_DOWN)) << 2,
+                        (R_ITEM_BTN_Y(HUD_ITEM_BTN_C_DOWN) + R_ITEM_BTN_WIDTH(HUD_ITEM_BTN_C_DOWN)) << 2,
+                        G_TX_RENDERTILE, 0, 0, R_ITEM_BTN_DD(HUD_ITEM_BTN_C_DOWN) * 2,
+                        R_ITEM_BTN_DD(HUD_ITEM_BTN_C_DOWN) * 2);
 
     // C-Right Button Color & Texture
     gDPSetPrimColor(OVERLAY_DISP++, 0, 0, R_C_BTN_COLOR(0), R_C_BTN_COLOR(1), R_C_BTN_COLOR(2),
                     interfaceCtx->cRightAlpha);
-    gSPTextureRectangle(OVERLAY_DISP++, R_ITEM_BTN_X(3) << 2, R_ITEM_BTN_Y(3) << 2,
-                        (R_ITEM_BTN_X(3) + R_ITEM_BTN_WIDTH(3)) << 2, (R_ITEM_BTN_Y(3) + R_ITEM_BTN_WIDTH(3)) << 2,
-                        G_TX_RENDERTILE, 0, 0, R_ITEM_BTN_DD(3) * 2, R_ITEM_BTN_DD(3) * 2);
+    gSPTextureRectangle(
+        OVERLAY_DISP++, R_ITEM_BTN_X(HUD_ITEM_BTN_C_RIGHT) << 2, R_ITEM_BTN_Y(HUD_ITEM_BTN_C_RIGHT) << 2,
+        (R_ITEM_BTN_X(HUD_ITEM_BTN_C_RIGHT) + R_ITEM_BTN_WIDTH(HUD_ITEM_BTN_C_RIGHT)) << 2,
+        (R_ITEM_BTN_Y(HUD_ITEM_BTN_C_RIGHT) + R_ITEM_BTN_WIDTH(HUD_ITEM_BTN_C_RIGHT)) << 2, G_TX_RENDERTILE, 0, 0,
+        R_ITEM_BTN_DD(HUD_ITEM_BTN_C_RIGHT) * 2, R_ITEM_BTN_DD(HUD_ITEM_BTN_C_RIGHT) * 2);
```

Comment:
Can these all do shifting or does that not work? If they can it should be consistent.

Reply:
Hm what shift is * 2?

Reply:
log2(2) is 1

Reply:
> Hm what shift is * 2?

`X << 1`